riscv-pages/src/sequential_pages.rs

```diff
@@ -39,7 +39,7 @@ pub struct SequentialPages<S: State> {
 }
```

Comment:
I'm all for more tests, but these changes are completely unrelated from the rest of the commit that deals with mapping hugepages in a page table and so should go in their own commit.